pytorch/notebooks/Serving PyTorch Models with CMLE  Custom Prediction Code.ipynb

```diff
@@ -0,0 +1 @@
CKET}/{GCS_MODEL_DIR} \\\n","            --runtime-version={RUNTIME_VERSION} \\\n","            --framework='SCIKIT_LEARN' \\\n","            --python-version=2.7 \\\n","            --package-uris=gs://{BUCKET}/{GCS_PACKAGE_URI}\\\n","            --model-class={MODEL_CLASS}"],"execution_count":0,"outputs":[]},{"metadata":{"id":"naG7FtQokRzM","colab_type":"code","colab":{"base_uri":"https://localhost:8080/","height":51},"outputId":"2df0240e-1445-47e8-9a41-627dfacab273","executionInfo":{"status":"ok","timestamp":1538246126033,"user_tz":-60,"elapsed":3411,"user":{"displayName":"Khalid Salama","photoUrl":"","userId":"16897751573918541504"}}},"cell_type":"code","source":["!gcloud ml-engine versions list --model {MODEL_NAME}"],"execution_count":24,"outputs":[{"output_type":"stream","text":["NAME  DEPLOYMENT_URI                                            STATE\n","v1    gs://ksalama-gcs-cloudml/models/pytorch/iris_classifier/  READY\n"],"name":"stdout"}]},{"metadata":{"id":"TQz-JaqpkRzP","colab_type":"text"},"cell_type":"markdown","source":["# D. CMLE Online Prediction"]},{"metadata":{"id":"Wdx5nTxjkRzR","colab_type":"code","colab":{}},"cell_type":"code","source":["from googleapiclient import discovery\n","from oauth2client.client import GoogleCredentials\n","\n","credentials = GoogleCredentials.get_application_default()\n","api = discovery.build('ml', 'v1', credentials=credentials,\n","                      discoveryServiceUrl='https://storage.googleapis.com/cloud-ml/discovery/ml_v1_discovery.json')\n","\n","\n","def estimate(project, model_name, version, instances):\n","    \n","    request_data = {'instances': instances}\n","\n","    model_url = 'projects/{}/models/{}/versions/{}'.format(project, model_name, version)\n","    response = api.projects().predict(body=request_data, name=model_url).execute()\n","\n","    #print response\n","    \n","    predictions = response[\"predictions\"]\n","    return predictions"],"execution_count":0,"outputs":[]},{"metadata":{"id":"ETKlF4yHkRzT","colab_type":"code","colab":{"base_uri":"https://localhost:8080/","height":34},"outputId":"f1089ffd-c6c0-4db9-cf64-02bf414a5701","executionInfo":{"status":"ok","timestamp":1538246147364,"user_tz":-60,"elapsed":869,"user":{"displayName":"Khalid Salama","photoUrl":"","userId":"16897751573918541504"}}},"cell_type":"code","source":["instances = [\n","    [6.8, 2.8, 4.8, 1.4],\n","    [6. , 3.4, 4.5, 1.6]\n","]\n","\n","predictions = estimate(instances=instances\n","                     ,project=PROJECT\n","                     ,model_name=MODEL_NAME\n","                     ,version=MODEL_VERSION)\n","\n","print(predictions)"],"execution_count":27,"outputs":[{"output_type":"stream","text":["[u'versicolor', u'versicolor']\n"],"name":"stdout"}]},{"metadata":{"id":"nk8oZ2zPkRzW","colab_type":"text"},"cell_type":"markdown","source":["# Questions? Feedback?\n","Feel free to send us an email (cloudml-feedback@google.com) if you run into any issues or have any questions/feedback!"]}]}
```

Comment:
Is it possible have this in multiple lines to make the review easier? 
When using Jupyter notebooks they usually spit out on multi-lines, but is this from colab?

Reply:
That's my plan, just making comments for the PR is a bit tricky.